Ports/mrsh/package.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=mrsh
 version=cd3c3a48055ab4085d83f149ff4b4feba40b40cb
-files="https://codeload.github.com/emersion/mrsh/legacy.tar.gz/${version} emersion-mrsh-d9763a3.tar.gz 6896493a1020774715ccca28e8d8f4ec722af63a93543fb6dd2762f7b1de9c8a"
+files="https://api.github.com/repos/emersion/mrsh/tarball/${version} ${port}-${version}.tar.gz 03fa30993c8cc9eab4b10092f278ace847de60484ac2ea21537c2b1922f09f9e"
```

Comment:
Can't we just use the well-known `archive` links, i.e. `https://github.com/emersion/mrsh/archive/${version}.tar.gz`?